### ISSUE TITLE
[Recommendation] Proper Semantic Versioned Releases

### ISSUE THREAD

**Issue by bigtimebuddy** [lerna/lerna#670]
First, thank you for such a wonderful tool.

Would it be possible to move the next releases away from pre-released beta tags "2.0.0-beta.x" and back to major, minor, patch releases? Couple of reasons why this would be important: 

* it would _really_ help to identify the scope of changes of releases
* would make it easier to fuzzy-version within projects
* the library has grown in stability and features over the last few months and is at or exceeded a v2 release
* the [v2.0.0 milestone](https://github.com/lerna/lerna/milestone/1) has no pending items in it

What else is planned for v2.0.0? Would love to see a major release and or release-candidates that get us closer to getting away from relatively-stable monthly pre-releases. 

Would love to hear the teams thoughts on this.

**Comment by bigtimebuddy** [comment]
Alternatively, you might consider using npm's [`dist-tag`](https://docs.npmjs.com/cli/dist-tag) command for managing canary, beta, latest, legacy releases. Would allow a developer to `npm install lerna@beta` to grab the latest beta release.

**Comment by evocateur** [comment]
I am 1000% in support of this.

I would like to loosen the [init version check](https://github.com/lerna/lerna/blob/master/src/Command.js#L116) to use `semver.satisfies` instead of the current exact comparison, but otherwise feel like we could cut over to 2.0.0 very soon.

**Comment by evocateur** [comment]
This is good context for #583

**Comment by bcoe** [comment]
@evocateur related request; as you march towards a `2.x` release (which I know can take a while, it took us a couple months to get yargs 7.x out the door) would it be possible to occasionally do:

`npm publish --tag=next-2`

Selfishly, I can then start using some of the new features for the monorepos I'm working on; less-selfishly, I can provide QA and feedback 😛

**Comment by evocateur** [comment]
@bcoe I think that's a fair request. I floated the idea today of "graduating" to `-rc` versions in our next publish, with a solid semver to follow soon after. My goal for tomorrow is to get some sort of prerelease out the door.

Meanwhile, I _really_ want to automate a `--canary` build from `master` so folks whose contributions we accept can start using them immediately (with full caveat of "nightly" status). I think we've got a couple things to do first before that happens.
